pkgs/development/python-modules/pyright/default.nix

```diff
@@ -0,0 +1,54 @@
+{
```

Comment:
I don't think this package is needed to build crewai, and it is causing many packages to rebuild.
Could you remove this package?

Reply:
Ok, removed pyright.

Reply:
You also need to remove it as a dependency